site.yml

```diff
@@ -13,7 +13,7 @@
     - role: app-user
 
     - role: rbenv
-      ruby_version: 2.3.7
+      ruby_version: "{{ lookup('file', 'fairfood/.ruby-version') }}"
```

Comment:
The first time we run this, the repository is not checked out yet and this will fail. I'm wondering if we can remove the version because the post-receive hook is now calling `./script/rbenv-install.sh`. If it needs a version, maybe we can just set it to `system`?

Reply:
🤦  Good pickup. 
Yes we shouldn't need this, I'll try removing.

Reply:
Replaced this commit with: 4a0b4ac
Tested on fresh vagrant 👍